fix(api) treat bigInt as string all across the system (#20888)
directus/directus: .changeset/small-poems-deny.md

---
'@directus/api': minor
---

> Fixed issue of not being able to insert or filter bigInt fields in graphQL api

Reviewer: ```suggestion
Fixed issue of not being able to insert or filter big integer fields via GraphQL
```
